FRINXio/frinx-frontend
# add/remove/clear nodes to common node search

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FD-403` 
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  | No


this is initial work on frontend, where user is able to add and remove nodes, which will be used for common search (endpoint will be added to frinx-inventory-server)

HOW TO TEST:

you need to run it against inventory server with this changes https://github.com/FRINXio/frinx-inventory-server/pull/229

run against .12 and open Topology discovery screen. by holding left or right shift + click on the nodes you should be able to add/remove nodes to react state, which will be later used for sending request to topology discovery api.
